lib/src/view/analysis/analysis_board.dart

```diff
@@ -17,14 +17,14 @@ class AnalysisBoard extends ConsumerStatefulWidget {
   const AnalysisBoard(
     this.options,
     this.boardSize, {
-    this.borderRadius,
+    this.radius,
     this.enableDrawingShapes = true,
     this.shouldReplaceChildOnUserMove = false,
   });
 
   final AnalysisOptions options;
   final double boardSize;
-  final BorderRadiusGeometry? borderRadius;
```

Comment:
Are you talking only about the name of the variable or the type ?

Reply:
both

Reply:
To add rounded corners to the players widgets instead of the board for broadcast like on the website

![image](https://github.com/user-attachments/assets/3518691a-6c27-4a9a-baa8-0b37b4d8c459)

The change of the name is just to match the type